server/routes/listingRouter.js

```diff
@@ -1,45 +1,76 @@
 const listingRouter = require('express').Router();
-// const { Listing } = require('../models.js'); <- uncomment when Listing model is implemented
+const { Listing } = require('../models.js');
 const { passport } = require('../jwtEncrypt.js');
 
-
-// Create
+// Create listing
 listingRouter.post('/', passport.authenticate('jwt', { session: false }), async (req, res) => {
-	try{
+	try {
+		let {
```

Comment:
Don't worry about this now, but in general if we don't need a variable to change, we want to declare it with `const`